Website/DesktopModules/Hotcakes/Hotcakes.dnn

```diff
@@ -58,7 +58,7 @@
 						<attributes>
 							<businessControllerClass>Hotcakes.Modules.Core.HotcakesController, Hotcakes.Modules</businessControllerClass>
 							<desktopModuleID>[DESKTOPMODULEID]</desktopModuleID>
-							<upgradeVersionsList>01.00.00,01.00.07,01.05.00,01.06.00,01.07.00,01.08.00,01.09.00,01.10.00,02.00.00,02.00.01,02.00.02,02.00.03,02.00.04,02.00.05,02.00.06,03.00.00,03.00.01,03.01.00,03.02.00,03.02.01,03.02.02,03.02.03,03.03.00,03.04.00,03.04.01,03.05.00,03.06.00,03.07.00,03.08.00,03.08.01,03.08.02,03.09.00</upgradeVersionsList>
+							<upgradeVersionsList>01.00.00,01.00.07,01.05.00,01.06.00,01.07.00,01.08.00,01.09.00,01.10.00,02.00.00,02.00.01,02.00.02,02.00.03,02.00.04,02.00.05,02.00.06,03.00.00,03.00.01,03.01.00,03.02.00,03.02.01,03.02.02,03.02.03,03.03.00,03.04.00,03.04.01,03.05.00,03.06.00,03.07.00,03.08.00,03.08.01,03.08.02,03.09.00,03.09.01</upgradeVersionsList>

```

Comment:
@ArielBlanco1990 This update is going to be for the 3.9.0 release, not 3.9.1.

Reply:
@WillStrohl This is now resolved.